ci-operator/config/openshift/cluster-kube-scheduler-operator/openshift-cluster-kube-scheduler-operator-master.yaml

```diff
@@ -40,3 +40,11 @@ tests:
   commands: TEST_SUITE=openshift/smoke-4 run-tests run-tests
   openshift_installer:
     cluster_profile: aws
+- as: e2e
+  commands: make e2e
+  container:
+    from: bin
```

Comment:
please make this job like this https://github.com/openshift/release/blob/master/ci-operator/config/openshift/machine-api-operator/openshift-machine-api-operator-master.yaml#L53-L56

these are going to be e2e and thus need a full cluster

Reply:
@sjenning updated PTAL. Though it seems to me that it will not create a full cluster for non-existing e2e tests.